spec/serializers/board_serializer_spec.rb

```diff
@@ -23,11 +23,15 @@
     expect(subject['created_at']).to be_present
   end
 
-  it 'makes json with updated at' do
-    expect(subject['updated_at']).to be_present
+  it 'makes json with users' do
```

Comment:
Why did we stop testing the `updated_at` field? :)

Reply:
The `updated_at` field was deleted in `BoardSerializer`